tests/components/season/test_sensor.py

```diff
@@ -16,17 +16,17 @@
 from homeassistant.setup import async_setup_component
 
 HEMISPHERE_NORTHERN = {
-    "homeassistant": {"latitude": "48.864716", "longitude": "2.349014"},
+    "homeassistant": {"latitude": 48.864716, "longitude": 2.349014},
```

Comment:
According to config type hint, it should be a float so the tests shouldn't be inserting a string.